src/components/Label/Label.tsx

```diff
@@ -25,27 +25,27 @@ export interface LabelProps
     ContentComponentProps {
   accessibility?: Accessibility
 
-  /** A label can be circular. */
+  /** A Label can be circular. */
   circular?: boolean
 
-  /** A Label can take the width of its container. */
+  /** A Label can take up the width of its container. */
   fluid?: boolean
 
-  /** Label can have an icon. */
+  /** A Label can have an icon. */
   icon?: ShorthandValue
 
-  /** An icon label can format an Icon to appear before or after the text in the label */
+  /** A Label can position its Icon at the start or end of the layout. */
```

Comment:
Make terminology consistent. Allowed prop values are "start" and "end", so we should use the same wording in the description.